doc/rules/function_notation/create_from_format_call.rst

```diff
@@ -0,0 +1,18 @@
+================================
+Rule ``create_from_format_call``
+================================
+
+The first argument of function ``createFromFormat`` must start with ``!``.
```

Comment:
First of all, thanks for raising the PR!

would you mind to add a note here "why" it must start with `!` ? like, imagine John Doe wondering if they should follow this rule or not, they have no clue why they should.